src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+
+>**Note**: Background adjustments such as centered, scaled, spanned, stretched, and zoom can be applied using the GNOME Tweaks application: [GNOME Tweaks Appearance Settings](gnome-tweaks-extensions.md#appearance)
+
+![Background Tab](/images/customize-desktop/background-tab.png)
+
+### Set Pre-installed Images as Wallpaper
+
+Select an image from the list. There is no “apply” button; the desktop background will update automatically when an image is selected.
+
+![Set Pre-installed Image](/images/customize-desktop/set-preinstalled-image.png)
+
+### Set User Images as Wallpaper
+
+1. Click `Add Picture…` and navigate to the location of your image.
+
+    ![Click Add Picture](/images/customize-desktop/click-add-picture.png)
+
+2. Click `Open`.
+
+    ![Click Open](/images/customize-desktop/click-open.png)
+
+3. Select the image added to the list of available wallpapers.
+
+    ![Select Added Image](/images/customize-desktop/select-added-image.png)
+
+You can also right-click the image and choose `Set as Wallpaper` from within the file browser.
+
+![Set as Wallpaper](/images/customize-desktop/set-as-wallpaper.png)
+
+## Appearance
+
+Pop!\_OS includes two default themes: Light and Dark. These themes apply to all system and application windows across the operating system. Selecting a theme will also assign a default desktop background.
+
+**Light Theme**
+
+![Light Theme](/images/customize-desktop/light-theme.png)
+
+**Dark Theme**
+
+![Dark Theme](/images/customize-desktop/dark-theme.png)
+
+Enable the Light or Dark theme in the `Appearance` tab. Additional theme customizations can be applied using [GNOME Tweaks](gnome-tweaks-extensions.md).
+
+![Change Theme](/images/customize-desktop/change-theme.png)
+
+## Dock
+
+The `Dock` tab contains settings for modifying Dock visibility behavior and the appearance of system and application icons.
+
+![Modify Dock](/images/customize-desktop/customize-dock.png)
+
+| Setting | Function |
+|----------|----------|
+| Enable Dock | Disabling this option causes the Dock to disappear entirely. |
+| Dock Options | Display utility icons and mounted drives in the Dock, make the Dock appear as a bar that extends to the screen edges, and specify what happens when an application icon is clicked. See examples of Icon Click Action behavior in [Navigate Between Running Applications](/navigate-pop/switching-apps.md#using-the-dock). <ul><li>Launch or Cycle Windows: A running application will be focused when its icon is clicked. Additional clicks will cycle through instances of that application.</li><li>Launch or Minimize Windows: A running application will be focused or minimized when its icon is clicked.</li><li>Launch, Minimize, or Preview Windows: When clicked, a running application will be minimized, focused, or the user can preview multiple windows (only if the application has multiple windows open).  |
```

Comment:
`a bar that extends to the screen edges` could be `a panel that extends to the screen edges` since the word for such a bar is typically "panel," but that's just a suggestion.

> When clicked, a running application will be minimized, focused, or the user can preview multiple windows (only if the application has multiple windows open).

The other two options say `will be`, this one includes a `can` and is therefore indeterminate (it sort of makes it sound like they have a choice in the matter beyond just selecting this option, which isn't the case.) Consider rewording in such a way that the reader will be more sure of what's going to happen:

> When clicked, if a running application has multiple windows open, a menu will be displayed with a preview of each window and the selected window will be focused. Otherwise, a running application will be focused or minimized.

Reply:
This is resolved by converting these all to imperative sentences, but I'm wondering if this part of the table should just become a header section with subsections and images for each option. I initially felt these options were self-explanatory and images would be overkill.

Reply:
> This is resolved by converting these to imperative sentences,

"...a menu displays with a preview of each window and focus applied to the selected window" is grammatically misleading, it sounds like focus will already be applied somehow when the menu displays. "focus is applied" would be less confusing and match the following sentence. (So just add the word "is," please.)

> I'm wondering if this part of the table should become a header section with subsections and images for each option.

It definitely could be if you wanted to. You might also try just adding some very small images in the table underneath each bullet point to break up the wall of text (e.g. an image of just an extended dock, an image of just the Launcher icon with the others hidden, an image of just the Workspaces icon with the others hidden, etc.)

Reply:
This section has been split out into headers and the suggested grammatical change has been applied.